A3A/addons/core/functions/FactoryOverhaul/fn_productionGUIControls.sqf

```diff
@@ -0,0 +1,186 @@
+//#include "fn_productionHandler.sqf"
+#include "fn_productionUnlocks.sqf"
```

Comment:
Generally ```#include``` just means "paste the contents of this file here", this doesn't really sound like something you want to be doing, but I'm not familiar enough with SQF to be completely certain that this is not basically what the interpreter does anyway. I'd lean towards not doing this, though.